RecoLocalCalo/HGCalRecProducers/python/hgcalLayerClusters_cff.py

```diff
@@ -6,30 +6,34 @@
 
 from RecoLocalCalo.HGCalRecProducers.HGCalUncalibRecHit_cfi import HGCalUncalibRecHit
 
-from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import fC_per_ele, hgceeDigitizer, hgchebackDigitizer, hfnoseDigitizer
+from SimCalorimetry.HGCalSimProducers.hgcalDigitizer_cfi import fC_per_ele, HGCAL_noises, hgceeDigitizer, hgchebackDigitizer, hfnoseDigitizer
 
 hgcalLayerClusters = hgcalLayerClusters_.clone()
```

Comment:
May I suggest the possibly cleaner (I hope the syntax is still correct...):
```suggestion
hgcalLayerClusters = hgcalLayerClusters_.clone(
    timeOffset = hgceeDigitizer.tofDelay,
    plugin = dict(
        dEdXweights = dEdX.weights,
        #With the introduction of 7 regional factors (6 for silicon plus 1 for scintillator),
        #we extend fcPerMip (along with noises below) so that it is guaranteed that they have 6 entries.
        fcPerMip = HGCalUncalibRecHit.HGCEEConfig.fCPerMIP + HGCalUncalibRecHit.HGCHEFConfig.fCPerMIP,
        thicknessCorrection = HGCalRecHit.thicknessCorrection,
        sciThicknessCorrection = HGCalRecHit.sciThicknessCorrection,
        deltasi_index_regemfac = HGCalRecHit.deltasi_index_regemfac,
        fcPerEle = fC_per_ele,
        #Extending noises as fcPerMip, see comment above.
        noises = HGCAL_noises.values + HGCAL_noises.values,
        noiseMip = hgchebackDigitizer.digiCfg.noise
    )
)
```